d3a_interface/sim_results/simulation_assets_info.py

```diff
@@ -0,0 +1,68 @@
+from typing import Dict, List
+
+from d3a_interface.sim_results import is_finite_power_plant_node_type, is_prosumer_node_type
+from d3a_interface.sim_results.results_abc import ResultsBaseClass
+from d3a_interface.utils import scenario_representation_traversal
+
+
+class SimulationAssetsInfo(ResultsBaseClass):
+    FIELDS = ['number_of_house_type', 'avg_assets_per_house', 'number_of_load_type',
+              'total_energy_demand_kwh', 'number_of_pv_type', 'total_energy_generated_kwh',
+              'number_of_storage_type', 'total_energy_capacity_kwh', 'number_of_power_plant_type',
+              'max_power_plant_power_kw']
+
+    def __init__(self):
+        self.assets_info = {field: 0 for field in SimulationAssetsInfo.FIELDS}
+
+    @staticmethod
+    def merge_results_to_global(market_results: Dict, global_results: Dict, slot_list: List):
+        pass
+
+    def update(self, area_result_dict, core_stats, current_market_slot):
```

Comment:
```suggestion
    def update(self, area_result_dict: Dict, core_stats: Dict, current_market_slot: str):
```
Since parameter type hint is already added in other methods of this module.